src/Features/VisualBasic/Completion/CompletionProviders/CompletionListTagCompletionProvider.vb

```diff
@@ -14,6 +14,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
         Inherits EnumCompletionProvider
 
         Protected Overrides Function GetPreselectedSymbolsWorker(context As AbstractSyntaxContext, position As Integer, options As OptionSet, cancellationToken As CancellationToken) As Task(Of IEnumerable(Of ISymbol))
+            If context.SyntaxTree.IsObjectCreationTypeContext(position, cancellationToken) Then
+                Return SpecializedTasks.EmptyEnumerable(Of ISymbol)()
+            End If
+
+
```

Comment:
What is this change here for?  No tests cover this or the VB change to `InferTypesWorker` for VB.